genie-docs/src/docs/asciidoc/_properties.adoc

```diff
@@ -230,6 +230,21 @@ Health of the system is marked unhealthy if the CPU load of a system goes beyond
 |null
 |yes
 
+|genie.jobs.attachments.location-prefix
+|Common prefix where attachments are stored
+|s3://genie/attachments
+|no
+
+|genie.jobs.attachments.max-size
+|Maximum size of an attachment
+|100MB
+|no
+
+|genie.jobs.attachments.max-total-size
+|Maximum size of all attachments combined (Spring and Tomcat may also independently limit the size of upload)
```

Comment:
This is true that until now we've used 
```
spring.servlet.multipart.max-file-size | 1MB | Max file size.
spring.servlet.multipart.max-request-size | 10MB | Max request size.
```
from [here](https://docs.spring.io/spring-boot/docs/current/reference/html/appendix-application-properties.html#web-properties).

With the addition of these properties someone would now have to tune 4 different properties and make them line up or get confused when it doesn't behave how they want (e.g. spring lets it through but then genie errors or vice versa). 

Curious what are you trying to accomplish by adding these additional properties?

Reply:
More flexible configuration.
Rather than relying on the underlying server limit, this limits specifically the attachments.
For example one may not want to touch the upload limit, but still limit attachments to 5MB. This is now possible.

Tuning 4 different properties is not necessary. One can keep the spring/tomcat (maybe not even know they exist) and limit the size of attachments with a simple clearly-named property.